pkg/jaeger/query/trace_query.go

```diff
@@ -237,8 +240,22 @@ func (b *Builder) buildTraceIDSubquery(q *spanstore.TraceQueryParameters) (strin
 				clauses = append(clauses, qual)
 			default:
 				//TODO make sure this is optimized correctly
-				params = append(params, k, "\""+v+"\"")
-				qual := fmt.Sprintf(`_ps_trace.tag_map_denormalize(s.span_tags)->$%d = $%d`, len(params)-1, len(params))
+				val := "\"" + v + "\""
+				if digitCheck.MatchString(v) {
```

Comment:
Do we make sure that value is trimmed? Otherwise we will not match: ` 1`, `1 ` or ` 1 `

Reply:
Also it seems we will not match `float` so I believe regex should be fixed

Reply:
Have you maybe tested what happens with booleans?

Reply:
I don't think trimming will be good. We should use it as we get from jaeger.